test/test/helpers/use-nock/README.md

```diff
@@ -18,7 +18,7 @@ useNock();
 ```js
 import useNock, { nock } from 'test/helpers/use-nock';
 //or
-import { useNock, nock } from 'text/helpers/use-nock';
+import { useNock, nock } from 'test/helpers/use-nock';
```

Comment:
I like the part of comment focusing on outside developer experience. Taking that into account we should rather go the other way around and update the code to use `nock` explicitly :)